pjsip/src/pjsip/sip_parser.c

```diff
@@ -1783,6 +1789,13 @@ static void parse_generic_array_hdr( pjsip_generic_array_hdr *hdr,
 }
 
 /* Parse generic array header. */
+static void parse_generic_array_hdr( pjsip_generic_array_hdr *hdr,
```

Comment:
This is pre-existing. I don't know why git interprets it as a new line, rather than a move of an old line to a new place. Be that as it may, I'm hesitant to change it since it was already in pjsip.